biz.aQute.resolve/src/biz/aQute/resolve/obr/plugin/ObrExporter.java

```diff
@@ -0,0 +1,85 @@
+package biz.aQute.resolve.obr.plugin;
+
+import static aQute.bnd.osgi.Constants.RESOLVE_EXCLUDESYSTEM;
+
+import java.io.ByteArrayOutputStream;
+import java.io.File;
+import java.util.AbstractMap.SimpleEntry;
+import java.util.Collection;
+import java.util.List;
+import java.util.Map;
+import java.util.Map.Entry;
+
+import org.osgi.resource.Resource;
+
+import aQute.bnd.annotation.plugin.BndPlugin;
+import aQute.bnd.build.Project;
+import aQute.bnd.osgi.FileResource;
+import aQute.bnd.osgi.repository.XMLResourceGenerator;
+import aQute.bnd.service.export.Exporter;
+import aQute.lib.exceptions.Exceptions;
+import biz.aQute.resolve.Bndrun;
+import biz.aQute.resolve.RunResolution;
+
+@BndPlugin(name = "OBR Exporter")
+public final class ObrExporter implements Exporter {
+
+    static final String TYPE = "exporter.obr";
+
+    @Override
+    public String[] getTypes() {
+        return new String[] { TYPE };
+    }
+
+    @Override
+    public Entry<String, aQute.bnd.osgi.Resource> export(
+            final String type,
+            final Project project,
+            final Map<String, String> options) throws Exception {
+
+        final String outputdir       = options.get("outputdir");
+        final String name            = options.getOrDefault("name", removeExtension(project.getName()) + ".xml");
+        final String excludesystem   = options.getOrDefault("excludesystem", "true");
+
+        File outputDirectory = null;
+        if (outputdir == null) {
+            outputDirectory = project.getTargetDir();
+        } else {
+            outputDirectory = new File(project.getBase(), outputdir);
```

Comment:
I think you want `project.getFile(outputdir)`. This will use the project's base.

Reply:
Good to know. will do.

Reply:
DONE ✔️

Reply:
DONE ✔️